package/cloudshell/email/email_service.py

```diff
@@ -137,3 +176,22 @@ def _load_and_format_template(self, template_name, **extra_args):
             raise Exception('Failed loading email template')
 
         return content
+
+    def validate_email_config(self) -> None:
+        """Validates SMTP server configuration.
+
+        Returns:
+            None
+        Raises:
+            smtplib.SMTPHeloError
+            smtplib.SMTPAuthenticationError
+            smtplib.SMTPNotSupportedError
+            smtplib.SMTPException
+            RuntimeError
+        """
+        try:
+            smtp = self._login()
+            smtp.close()
+        except (smtplib.SMTPHeloError, smtplib.SMTPAuthenticationError,
+                smtplib.SMTPNotSupportedError, smtplib.SMTPException, RuntimeError):
+            raise
```

Comment:
this "except" statement is not doing anything and looks to be redundant. And since "_login" handles all the errors I think we can probably remove it?

Reply:
removed redundant statement